Userland/Libraries/LibGUI/AutocompleteProvider.cpp

```diff
@@ -100,6 +100,14 @@ AutocompleteBox::AutocompleteBox(TextEditor& editor)
     m_suggestion_view = main_widget.add<GUI::TableView>();
     m_suggestion_view->set_column_headers_visible(false);
     m_suggestion_view->set_visible(false);
+    m_suggestion_view->on_activation = [&](GUI::ModelIndex const& index) {
+        if (m_suggestion_view->model()->is_within_range(index)) {
+            m_suggestion_view->selection().set(index);
+            m_suggestion_view->scroll_into_view(index, Orientation::Vertical);
+        }
+
+        apply_suggestion();
```

Comment:
can this be done inside the if? If so, the if could be inverted (`!is_within_range()`) and do an early return for less nesting.

Reply:
Thank you for pointing me out the new guidelines :)
Now all should be fine